src/measure_theory/integral/integrable_on.lean

```diff
@@ -563,3 +563,90 @@ lemma continuous_on.strongly_measurable_at_filter_nhds_within {α β : Type*} [m
   (hf : continuous_on f s) (hs : measurable_set s) (x : α) :
   strongly_measurable_at_filter f (𝓝[s] x) μ :=
 ⟨s, self_mem_nhds_within, hf.ae_strongly_measurable hs⟩
+
+section partial_order
```

Comment:
I think it would be nice to add a section comment along the lines of
```
/-! ### Lemmas about adding and removing interval boundaries

The primed lemmas take explicit lemmas about the measure being finite at the endpoint, while the unprimed ones use `has_no_atoms`
-/
```
I'm sure you can do a better job of wording it than I can.